testing/cli/test_cli_sct_propseg.py

```diff
@@ -43,8 +43,26 @@ def test_isct_propseg_compatibility():
 
 @pytest.mark.skipif(sys.platform.startswith("win32"), reason="sct_propseg is not supported on Windows")
 def test_sct_propseg_o_flag(tmp_path):
-    argv = ['-i', sct_test_path('t2', 't2.nii.gz'), '-c', 't2', '-ofolder', str(tmp_path), '-o', 'test_seg.nii.gz']
-    sct_propseg.main(argv)
-    output_files = sorted([f for f in os.listdir(tmp_path)
-                          if os.path.isfile(os.path.join(tmp_path, f))])
-    assert output_files == ['t2_centerline.nii.gz', 'test_seg.nii.gz']
+    sct_propseg.main(['-i', sct_test_path('t2', 't2.nii.gz'), '-c', 't2', '-ofolder', str(tmp_path),
+                      '-o', 'test_seg.nii.gz'])
+    output_files = set([f for f in os.listdir(tmp_path)])
```

Comment:
Nitpick: you can pass a generator expression directly instead of a list comprehension to `set()`, since it accepts any iterator. (Although possibly this is not the style we use in SCT?)
```suggestion
    output_files = set(f for f in os.listdir(tmp_path))
```

Reply:
Ah, good catch! 

I actually didn't realize "[set comprehensions](https://python-reference.readthedocs.io/en/latest/docs/comprehensions/set_comprehension.html)" were a a supported syntax in much the same way list comprehensions are. So, we can do this, too:

```suggestion
    output_files = {f for f in os.listdir(tmp_path)}
```